terrautils/metadata/lemnatec.py

```diff
@@ -1,24 +1,32 @@
-'''
-Create a cleaned/curated metadata object based on the LemnaTec fixed metadata
-Some metadata used for search
-Some metadata used for extractors
+"""
+Given a JSON object containing the original LemnaTec metadata, convert
+field names and values to standardized formats required for TERRA-REF.
 
-TODO: 
-  Add checks for expected fields
-  Date/time standardization
-  Add units to variables?
-  Logging
+The "cleaned" metadata will be posted to the Clowder metadata endpoint and
+used downstream by extractors.
 
-'''
+TODO:
+    SENSOR_IRRIGATION                                                                                   
+    SENSOR_LIGHTNING
+    SENSOR_ENVIRONMENTAL_LOGGER
+    SENSOR_WEATHER
+"""
 
+import argparse
 import json
+import logging
 import os
+import pytz, datetime
 import requests
-from terrautils import sensors
+import sys
+from terrautils.sensors import Sensors
+
+STATION_NAME = "ua-mac"
+TERRAREF_BASE='/projects/arpae/terraref/sites'
```

Comment:
I don't want to be tedious but is this something that could be defined in a single location? 

Mostly I want to make sure that it will be as easy as possible to transition this project from one filesystem to another (seems likely to happen, though probably not before Nov.)